### ISSUE TITLE
Add support for `jdk.ObjectAllocationOutsideTLAB` and `jdk.ObjectAllocationInNewTLAB` JFR events

### ISSUE THREAD

**Issue by roberttoyonaga** [oracle/graal#5636]
**TL;DR**
Add support for the`jdk.ObjectAllocationOutsideTLAB` and `jdk.ObjectAllocationInNewTLAB` JFR events.

**Goals**
These events should detail the object class and allocation size when an object is allocated outside or in a new TLAB.

When this work is done, the[ JFR Support tracking Issue](https://github.com/oracle/graal/issues/5410) should be updated to reflect the changes.

**Describe who do you think will benefit the most.**
GraalVM users will benefit from the additional information JFR will report. GraalVM developers will also benefit because object allocations outside TLAB events can help debug performance issues. 

**Express whether you'd like to help by contributing to this feature**
I volunteer to work on this.

**Comment by fniephaus** [comment]
#5654 is merged, does this mean this issue is done and we need to update #5410?
